Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -48,10 +48,10 @@ If this value is empty, each pod will get an ephemeral directory to store their
     * `enabled`: Whether to enable prometheus based monitoring for external or internal cluster
     * `externalMgrEndpoints`: external cluster manager endpoints
     * `externalMgrPrometheusPort`: external prometheus manager module port. See [external cluster configuration](#external-cluster) for more details.
-    * `rulesNamespace`: Namespace to deploy prometheusRule. If empty, namespace of the cluster will be used.
+    * `rulesNamespaceOverride`: Namespace to deploy prometheusRule. If empty, namespace of the cluster will be used.
```

Comment:
Actually this setting isn't in the cluster CRD anymore, it's only a helm setting, so we need to delete this and its sub-bullets

Reply:
Okay